tracks/java/exercises/hamming/mentoring.md

```diff
@@ -0,0 +1,47 @@
+### Reasonable solutions
+
+```java
+class Hamming {
+    private String leftStrand, rightStrand;
+    
+    Hamming(String leftStrand, String rightStrand) {
+        if (leftStrand.length() != rightStrand.length()) {
+            throw new IllegalArgumentException("leftStrand and rightStrand must be of equal length.");
+        this.leftStrand = leftStrand;
+        this.rightStrand = rightStrand;
+        }
```

Comment:
Probably need to move this to line 10.

Reply:
Right now (and this is unclear because of the faulty indentation), the assignment is inside the guard clause (with the `throw` right before the assignments).
```java
this.leftStrand = leftStrand;
this.rightStrand = rightStrand;
```

`this` refers to the bracket `}`

Reply:
I apologize for the mistake. That's what happen when you want to go too fast... I commited a fix for the solutions. Thanks for your watchfulness!